test/server/routes/applications.test.js

```diff
@@ -35,6 +35,95 @@ describe('routes/applications.js', () => {
     done();
   });
 
+  it('should serve up the applications index page on the /manage-applications mount path', () => {
+    const slug = '/manage-applications';
+    const stubGetApplicationList = sinon.stub(ApplicationsDeveloperService.prototype, 'getApplicationList')
+      .returns(Promise.resolve(serviceData.getApplicationList));
```

Comment:
Shouldn't this return serviceData. getList instead of . getApplicationList?

Reply:
it should , I will change it now :)

Reply:
Or should service data contain a 'getApplicationList' and the Api stuff gets move in there under 'getApiKeyList'?